src/test/external/docker-registry-values.yaml

```diff
@@ -15,5 +15,7 @@ service:
 
 # Note: Super fake and not real htpasswd for testing purposes
 secrets:
-  haSharedSecret: ""
   htpasswd: "push-user:$2a$10$bnke4DeqY4qsAWIRsJCFluayx4v56mxhp/bfwubt.K83fudDzKAue"
+  configData:
+    http:
+      secret: "q1EouKxQvHtNho3MAoLwFJHCz8Fcbc8E4XLzm!McQuwmHGJG"
```

Comment:
Note - this is a randomly generated string that is required for the zarf-registry helm chart. More important for the upgrade lifecycle in zarf but should be irrelevant here - setting to bypass the `required` helm chart value check.